andela/mmdp_site
# MMDP-569  Remove beneficiary form on Cancel

**What does this PR do?**

Implements remove beneficiary form on cancel

**description of Task to be completed?**

- [x] Add cancel button to the beneficiary form
- [x] Implement remove beneficiary form on cancel

**How should this be manually tested?**
- check out to the branch below
```
 $ git checkout ft-remove-beneficiary-form-569
```
- start the development server
```
$  yarn run dev-server
```
- Using your browser, navigate to the link below on the **CMS**
```
http://localhost:1234/stakeholder-directory/add-basic-information

```

**Any background context you want to provide?**
N/A
Any pertinent information that should be considered

**What are the relevant Jira issues?**

[MMDP-569](https://viisaus.atlassian.net/browse/MMDP-569)

**Screenshots (if appropriate)**

<img width="1266" alt="Screenshot 2019-05-22 at 09 58 36" src="https://user-images.githubusercontent.com/9401930/58153677-4fe77780-7c78-11e9-885a-cb7af74a5945.png">

**Questions:**
N/A
